CryptoScan.Web/Main/Common/Http.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System.Text.Json;
+
+namespace CryptoScan.Web.Main.Common;
+
+internal static class Http
```

Comment:
I don't like this approach because using new HttpClient is not mockable at all, HttpClientFactory with DI should be used instead in production code.
But since it's just a uni project... ;)